ft/tests/test_filter.py

```diff
@@ -1,27 +1,22 @@
-from mock_command import mock_command
+from mock_command import mock
 from ft.commands.filter import Filter
 
 
 def test_filter_const_false():
-    test_filter = mock_command(Filter)
-    test_filter.set_input("const", ["false"], ["3", "5"])
-    test_filter.assert_output([])
+    cmd = mock(Filter, ("const", ["false"], ["3", "5"]))
+    assert cmd.output() == []
 
 
 def test_filter_const_true():
-    test_filter = mock_command(Filter)
-    test_filter.set_input("const", ["true"], ["3", "5"])
-    test_filter.assert_output(["3", "5"])
+    cmd = mock(Filter, ("const", ["true"], ["3", "5"]))
+    assert cmd.output() == ["3", "5"]
```

Comment:
This produces very similar error messages to `nose.tools.assert_equals()` (as long as the assertion is made immediately within the test function, and `pytest` is run with `-vv`.

Also makes it somewhat clearer what's happening here, I'd say.